website/docs/r/lexv2models_slot_type.html.markdown

```diff
@@ -0,0 +1,151 @@
+---
+subcategory: "Lex V2 Models"
+layout: "aws"
+page_title: "AWS: aws_lexv2models_slot_type"
+description: |-
+  Terraform resource for managing an AWS Lex V2 Models Slot Type.
+---
+
+# Resource: aws_lexv2models_slot_type
+
+Terraform resource for managing an AWS Lex V2 Models Slot Type.
+
+## Example Usage
+
+### Basic Usage
+
+```terraform
+resource "aws_iam_role_policy_attachment" "test" {
+  role       = aws_iam_role.test.name
+  policy_arn = "arn:${data.aws_partition.current.partition}:iam::aws:policy/AmazonLexFullAccess"
+}
+
+resource "aws_lexv2models_bot" "test" {
+  name                        = "testbot"
+  idle_session_ttl_in_seconds = 60
+  role_arn                    = aws_iam_role.test.arn
+
+  data_privacy {
+    child_directed = true
+  }
+}
+
+resource "aws_lexv2models_bot_locale" "test" {
+  locale_id                        = "en_US"
+  bot_id                           = aws_lexv2models_bot.test.id
+  bot_version                      = "DRAFT"
+  n_lu_intent_confidence_threshold = 0.7
+}
+
+resource "aws_lexv2models_bot_version" "test" {
+  bot_id = aws_lexv2models_bot.test.id
+  locale_specification = {
+    (aws_lexv2models_bot_locale.test.locale_id) = {
+      source_bot_version = "DRAFT"
+    }
+  }
+}
+resource "aws_lexv2models_slot_type" "test" {
+  bot_id      = aws_lexv2models_bot.test.id
+  bot_version = aws_lexv2models_bot_locale.test.bot_version
+  name        = "test"
+  locale_id   = aws_lexv2models_bot_locale.test.locale_id
+}
+```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `bot_id` - Identifier of the bot associated with this slot type.
```

Comment:
```suggestion
* `bot_id` - (Required) Identifier of the bot associated with this slot type.
```
I like what you've got here better than our current standard: Dropping `(Required)` and `(Optional)` where they are communicated another way. Unfortunately, we've got all this documentation with Required/Optional on every line regardless.